src/main/java/org/kainos/ea/db/DeliveryProjectDao.java

```diff
@@ -0,0 +1,33 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.DeliveryProject;
+import org.kainos.ea.cli.DeliveryProjectRequest;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DeliveryProjectDao {
+
+    private static Connection conn;
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public void createDeliveryProject(DeliveryProjectRequest dp) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO Project_Delivery(Project_ID, Delivery_ID, Actively_Working) " +
+                "VALUES(?, ?, 1)";
```

Comment:
Looks like you ignored the value in the Actively_Working parameter! This normally wouldn't make it into main!